includes/class-main.php

```
@@ -143,6 +143,13 @@ private function includes(): void {
 		require_once WPPO_PLUGIN_PATH . 'includes/class-core-tweaks.php';
 		require_once WPPO_PLUGIN_PATH . 'includes/class-object-cache.php';
 
+		// Phase 1 — Local Diagnostics (v1.5.0).
+		// Only load on admin, AJAX, or REST requests — not on every frontend page load.
+		if ( is_admin() || wp_doing_ajax() || ( defined( 'REST_REQUEST' ) && REST_REQUEST ) ) {
+			require_once WPPO_PLUGIN_PATH . 'includes/class-telemetry.php';
```

PHPStan (2.1.46)

[warning] Constant WPPO_PLUGIN_PATH not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)